app/build.gradle.kts

```diff
@@ -42,7 +42,12 @@ android {
     buildTypes {
         debug {
             isDebuggable = true
```

Comment:
You can also add `applicationIdSuffix` (and maybe a different name for debug) so that we can have two versions in the phone: local build and playstore.

Reply:
We can have these changes with a different pr if that's okay..

Reply:
yes